src/app/fio.js

```diff
@@ -186,7 +185,7 @@ class ScoreStatTable {
   getAltFreqForVariant(variant) {
     let freq = this.altFreq[this.variantMap.get(variant)];
     if (freq == null) {
-      throw new Error("Variant did not exist when looking up alt allele freq: " + variant);
+      throw new Error(`Variant did not exist when looking up alt allele freq: ${  variant}`);
```

Comment:
It seems like the new linting sometimes changes template strings to `${ variable}` and other times to `${ variable }`. Any idea what might be causing that?

Reply:
Honestly not sure- I suspect it's a deeply arcane quirk of whatever AST transformations are used. This happened a bunch of times when I converted LocusZoom as well. Kind of annoying to clean up- once review is done I'll do a pass to standardize all of that.

Readability aside, using template strings should help avoid some of the [small quirks](https://stackoverflow.com/a/16522695/1422268) of the language around operator overloading/ "helpful" type coercion when writing pure JS.